src/core/env.js

```diff
@@ -8,13 +8,28 @@
 
 var env = {};
 
-if (typeof wx === 'object' && typeof wx.getSystemInfoSync === 'function') {
-    // In Weixin Application
+/**
+ * get mini app api
+ *
+ * @see https://smartprogram.baidu.com/docs/develop/api/device_sys/#swan-getSystemInfoSync/
+ * @see https://developers.weixin.qq.com/miniprogram/dev/api/wx.getSystemInfoSync.html
+ * @return {boolean} result
+ */
+function getMiniApp() {
+    var wx = wx || null;
+    var swan = swan || null;
```

Comment:
The global variable `wx` exists not only on wechat miniapp, but also on the other mini h5 pages, if they use the the js library provided by wechat.